1-WebApp-OIDC/1-4-Sovereign/README.md

```diff
@@ -24,43 +30,38 @@ Note that, enabling your application for sovereign clouds requires you to:
-
-There is one project in this sample. To register it, you can:
 
-- either use PowerShell scripts that **automatically** creates the Azure AD applications and related objects (passwords, permissions, dependencies) for you and modify the Visual Studio projects' configuration files. If you want to use this automation:
+### Step 1: Download/Clone this sample code 
+This sample was created from the dotnet core 2.2 [dotnet new mvc](https://docs.microsoft.com/dotnet/core/tools/dotnet-new?tabs=netcore2x) template with `SingleOrg` authentication, and then tweaked to let it support tokens for the Azure AD V2 endpoint.
 
-  1. On Windows run PowerShell and navigate to the solution's folder
-  2. In PowerShell run:
+You can clone this sample from your shell or command line:
 
-  ```PowerShell
-  Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope Process -Force
+  ```console
+git clone https://github.com/Azure-Samples/microsoft-identity-platform-aspnetcore-webapp-tutorial webapp
+cd webapp
+cd "1-WebApp-OIDC\1-1-MyOrg"
   ```
 
-  3. Run the script to create your Azure AD application and configure the code of the sample application accordinly
-
-  ```PowerShell
-  .\AppCreationScripts\Configure.ps1
-  ```
+> Given that the name of the sample is very long, and so are the name of the referenced NuGet packages, you might want to clone it in a folder close to the root of your hard drive, to avoid file size limitations on Windows.
 
-   > Other ways of running the scripts are described in [App Creation Scripts](./AppCreationScripts/AppCreationScripts.md)
+### Step 2: Register the sample with your Azure AD tenant
 
-  4. Open the Visual Studio solution and click start. That's it!
+1. Sign in to the [Azure portal](https://docs.microsoft.com/en-us/azure/active-directory/develop/authentication-national-cloud#app-registration-endpoints) of the National Cloud of your choice using either a work or school account.
 
-- or, if you don't want to use automation, follow the steps below:
+> Note: Azure Germany doesn't support **App registrations (Preview)* experience.
 
-#### Choose the Azure AD tenant where you want to create your applications
+2. Choose the Azure AD tenant where you want to create your applications
+      - If your account is present in more than one Azure AD tenant, select profile button at the top right corner in the menu on top of the page and select `Switch Directory`.
+      - On `Directory + Subscription` switch your portal session to the desired Azure AD tenant.
 
-1. Sign in to the [Azure portal](https://portal.azure.com) using either a work or school account or a personal Microsoft account.
-1. If your account is present in more than one Azure AD tenant, select `Directory + Subscription` at the top right corner in the menu on top of the page, and switch your portal session to the desired Azure AD tenant.
 1. In the left-hand navigation pane, select the **Azure Active Directory** service, and then select **App registrations (Preview)**.
```

Comment:
GA will happen all across, WW+NC.  About deep link, we are not going to support the new experience in Germany, as it is going to be deprecated. So I think it can only be used in case of a specific example like US Gov.